plugins/user_mapping/src/user_claim.cpp

```diff
@@ -6,27 +6,52 @@
 #include <optional>
 #include <string>
 
+#include <boost/regex.hpp>
 #include <nlohmann/json.hpp>
 #include <spdlog/spdlog.h>
 
 namespace
 {
 	// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 	std::string claim_to_match;
+	// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+	boost::basic_regex<char> match_regex;
```

Comment:
Just saw in the boost docs, we can change this to be `boost::regex` if we want. It's a `typedef` of the same thing.

```suggestion
	boost::regex match_regex;
```

Reply:
Sure. That aligns with the C++ std lib implementation too.